backend/src/Controller.ts

```diff
@@ -161,8 +160,7 @@ class Controller {
 
   clearAll = async () => {
     Logger.info('CLEAR CREDENTIALS')
-    await user.clearAll() // moved to before clearing connections because having many connection would cause sign out to fail.
-    await this.pool.clearAll()
+    await user.clearAll()
```

Comment:
why do you need `await`?

Reply:
No, don't need await... and this is actually changing again... I'll update...